examples/go/tagging/.vcconfig

```diff
@@ -0,0 +1,8 @@
+[vcenter]
+server = "10.0.0.1"
```

Comment:
In your case, does your fn handle the "/sdk" endpoint correctly if not specified?

Reply:
Yup--so, I'll have the user give me the server address (e.g. "10.0.0.1"), and I'll append the "/sdk" path with the help of the net/url library.